canonical-websites/snapcraft-flask
# Move homepage from snapcraft.io-static-pages

**Please review https://github.com/canonical-websites/snapcraft-flask/pull/58 first**

Since K8s won't let me do the routing quite as I wanted, it's easier to just move the homepage into this app.

Ultimately this app should serve the homepage anyway. The way it's done here isn't ideal, so I've filed https://github.com/canonical-websites/snapcraft-flask/issues/59 to do soon, and made it high priority.

QA
--

``` bash
./run
```

Go to <http://127.0.0.1:8022>, see that it looks identical to <https://snapcraft.io>.

Now go to <http://127.0.0.1:8022/lxd/> and see a snap details page. Also go to <http://127.0.0.1:8022/missing> and see a 404 page.
